src/routes/reference/reactive-utilities/on.mdx

```diff
@@ -38,7 +38,7 @@ setA("new") // now it runs
 <Callout>
 	Please note that on stores and mutable, adding or removing a property from the
 	parent object will trigger an effect. See{" "}
-	<a href="/reference/stores/store-utilities#createmutable">
+	<a href="/reference/store-utilities/create-mutable">
 		createMutable
 	</a>
```

Comment:
I know this was already like this before. But shouldn't we use a QuickLink here instead?